content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -166,7 +166,7 @@ If you are on Ubuntu or another Linux distribution that supports the [snap](http
 
 ```shell
 snap install kubectl --classic
-kubectl version --client
+kubectl version --short
```

Comment:
Why is this better? (It produces a warning now and will produce an error in the future).